spec/requests/api/person_escort_records_controller_update_spec.rb

```diff
@@ -6,6 +6,12 @@
   describe 'PATCH /person_escort_records/:person_escort_record_id' do
     include_context 'with supplier with spoofed access token'
 
+    subject(:patch_person_escort_record) do
+      patch "/api/v1/person_escort_records/#{person_escort_record_id}", params: person_escort_record_params, headers: headers, as: :json
```

Comment:
we're moving away from the versioned urls, use `/api/person_escort_records/#{person_escort_record_id}"` instead (it will go to the same place)